src/System.Management.Automation/DscSupport/ConvertCimMofToJsonCommand.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
I agree for the completely new code files like this one (ConvertCimMofToJsonCommand.cs). However this one is no longer there because the conversion cmdlet was moved to DSC module.